tests/bluetooth/bsim_bt/bsim_test_audio/src/mics_test.c

```diff
@@ -0,0 +1,494 @@
+/*
+ * Copyright (c) 2020-2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifdef CONFIG_BT_MICS
+#include "bluetooth/audio/mics.h"
+#include "common.h"
+
+extern enum bst_result_t bst_result;
+
+#if defined(CONFIG_BT_AICS)
+#define AICS_DESC_SIZE CONFIG_BT_AICS_MAX_INPUT_DESCRIPTION_SIZE
+#else
+#define AICS_DESC_SIZE 0
+#endif /* CONFIG_BT_AICS */
+
+static struct bt_mics mics;
+
+static volatile uint8_t g_mute;
+static volatile int8_t g_aics_gain;
+static volatile uint8_t g_aics_input_mute;
+static volatile uint8_t g_aics_mode;
+static volatile uint8_t g_aics_input_type;
+static volatile uint8_t g_aics_units;
+static volatile uint8_t g_aics_gain_max;
+static volatile uint8_t g_aics_gain_min;
+static volatile bool g_aics_active = true;
+static char g_aics_desc[AICS_DESC_SIZE];
+static volatile bool g_cb;
+static struct bt_conn *g_conn;
+static bool g_is_connected;
+
+static void mics_mute_cb(struct bt_conn *conn, int err, uint8_t mute)
+{
+	if (err != 0) {
+		FAIL("MICS mute cb err (%d)", err);
+		return;
+	}
+
+	g_mute = mute;
+
+	if (conn == NULL) {
+		g_cb = true;
+	}
```

Comment:
This will prevent g_cb from being set if this callback is being called for a client, I think.
(Although that should not happen here.)

Reply:
That is correct. The name of the variable might be slightly misleading, but it is only used in the server only test at this point. Setting it would of course not do any harm, but it is also unused.